pkg/cmd/component/component/delete.go

```diff
@@ -69,6 +69,10 @@ func initiateNonInteractiveModeDeleteComponent(io *DeleteCompParameters) error {
 	if !exists {
 		return fmt.Errorf("the given Application: %s doesn't exists in the Path: %s", io.ApplicationName, io.Output)
 	}
+	presentComponents := ui.NumberOfComponents(filepath.Join(io.Output, io.ApplicationName, "components"))
+	if len(presentComponents) == 0 {
+		return fmt.Errorf("there are no components in the %s application in folder: %s ", io.ApplicationName, io.Output)
+	}
 	exists, _ = ioutils.IsExisting(ioutils.NewFilesystem(), filepath.Join(io.Output, io.ApplicationName, "components", io.ComponentName))
```

Comment:
For `ioutils.NewFilesystem()`, could you pull this out and assign a variable that can be reused, eg. at line 107/111

Reply:
Yes. I have made the changes